src/language-js/print/array.js

```diff
@@ -75,11 +77,20 @@ function printArray(path, options, print) {
         return element[itemsKey] && element[itemsKey].length > 1;
       });
 
+    const shouldUseFill =
+      n.elements.length > 1 &&
```

Comment:
nit: no single letter identifiers. (again, prob unrelated
here, just doing a code review as if this were
one of our projects)

Reply:
`n` is used everywhere, maybe long history, but I agre we should fix them someday.

Reply:
It's definitely not something to be done in this PR though.